lua/luarocks-tag-release.lua

```diff
@@ -92,7 +93,19 @@ local function luarocks_tag_release(package_name, package_version, specrev, args
       .. ' --api-key $LUAROCKS_API_KEY'
       .. luarocks_extra_flags_and_args
     print('UPLOAD: ' .. cmd)
-    local stdout, _ = OS.execute(cmd, error, args.is_debug)
+    local stdout, _ = OS.execute(cmd, function(message)
+      if message:find('already exists on the server') and not args.fail_on_duplicate then
```

Comment:
suggestion: we should probably add an integration test for this (a workflow that uploads luarocks-tag-release with a hardcoded version), just in case the error message changes after a luarocks update.

Reply:
That's a good idea, I like that. Shall we do that in a separate PR or in this one?

Reply:
I'm fine with creating an issue for it and merging this one.